hey_fireball.py

```diff
@@ -80,6 +79,9 @@ def __init__(self, msg):
                 self.target_name = self.target_id
             self.command = self._extract_command()
             self.count = self._extract_count()
+            self.setting = self._extract_setting()
+            self.setting = self._extract_setting() # Find on/off or assume toggle
```

Comment:
This line is repeated.

Reply:
A few things like this happened last night. I was trying to pick and choose what to push, and made several errors. Sorted it out, and stopped using the computer immediately